include/taskolib/deserialize_sequence.h

```diff
@@ -96,13 +96,13 @@ std::istream& operator>>(std::istream& stream, Step& step);
 Step load_step(const std::filesystem::path& lua_file);
 
 /**
- * Load a step setup script into the Sequence.
+ * Load parameters into the Sequence.
```

Comment:
Maybe we could add a little more detail, e.g. "Load sequence parameters like the step setup script and the sequence timeout."